bin/docker-build.js

```diff
@@ -757,8 +938,10 @@ function importImageSnapshotsIntoImgapi(builder, opts, callback) {
         }
 
         log.debug('Creating image for build layer %d %j', idx, dockerImage);
-        builder.emitStdout(util.format('Importing image %s into '
-            + 'IMGAPI\n', builder.getShortId(dockerImage.id)));
+        if (builder.commandType !== 'commit') {
```

Comment:
No stdout messages should be generated for a docker commit... and this check should be moved to the actual `build.emitStdout` function.